applications/notifications/types.py

```diff
@@ -9,7 +9,7 @@ class NotificationType(TextChoices):
     )
     BERTH_APPLICATION_REJECTED = (
         "berth_application_rejected",
-        _("Berth application rejected"),
+        _("Berth application processed"),
```

Comment:
Let's keep this one as it is, but change the value on `get_email_subject`

Reply:
ok fixed